File: PRAK403-2310817220016-NailaHanifah.c
#include <stdio.h>

int main() {
    int angka1, angka2, x;

    printf("Input (dua angka dipisahkan spasi): ");
    scanf("%d %d", &angka1, &angka2);

    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; i++) {
            x = angka2;
            x = x - (i - angka1);
            printf("%d %d", i, x);
            if (i == angka2){
                break;
            } else{
                printf(" - ");
            }
        }
    } else {
        for (int i = angka1; i >= angka2; i--) {
            x = angka2;
            x = x + (angka1 - i);
            printf("%d %d", i, x);
            if (i == angka2){
                break;
            } else{
                printf(" - ");
            }
        }
    }

    return 0;
}

File: PRAK403-2310817220016-NailaHanifah.py
angka1, angka2 = map(int, input("Input (dua angka dipisahkan spasi): ").split())

if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        x = angka2
        x = x - (i - angka1)
        print(f"{i} {x}", end="")
        if i == angka2:
            break
        else:
            print(" - ", end="")
else:
    for i in range(angka1, angka2 - 1, -1):
        x = angka2
        x = x + (angka1 - i)
        print(f"{i} {x}", end="")
        if i == angka2:
            break
        else:
            print(" - ", end="")